### ISSUE TITLE
posting dead blind might be wrong

### ISSUE THREAD

**Issue by GO1111** [hippich/Bitcoin-Poker-Room#96]
Played my BB, then sat out for the small blind. When I came back after the button, I was forced to post both BB and SB. Also seems like you should be prompted whether you want to post in or pay missed blinds vs waiting.

**Comment by hippich** [comment]
Are you sure this is not "dead" blind? http://www.homepokertourney.com/button.htm

**Comment by GO1111** [comment]
no, cuz what happened is that I did pay my big blind but had to sit out the 
small blind cuz I got a phone call. then after the button past, I was forced to 
repay a 2nd big blind and also the small blind. I should have just paid the 
small blind that I missed.

---

From: hippich 
reply@reply.github.com
To: fuzzymedia@yahoo.com
Sent: Thu, July 21, 2011 5:45:47 PM
Subject: Re: [Bitcoin-Poker-Room] posting dead blind might be wrong (#96)

Are you sure this is not "dead" blind? 
http://www.homepokertourney.com/button.htm

##

**Comment by GO1111** [comment]
Here is another seemingly misposting where I ended up posting the SB twice in a row, here are the HH numbers
301298
301297
301296
